examples/PlanarWalker/test_contact.cc

```diff
@@ -148,13 +147,13 @@ int do_main(int argc, char* argv[]) {
   tree.get()->surfaceTangents(n_world, d_world);
 
   Eigen::Matrix<double,3,2> d;
-  surfaceTangents(normal.col(0),d);
+  //  surfaceTangents(normal.col(0),d);
```

Comment:
Why is this line commented out?

Reply:
The original method is being deprecated. I didn't think it was worth replicating the method to test a rigidbodytree function.